src/pgduckdb_types.cpp

```diff
@@ -236,6 +236,18 @@ ConvertTimeDatum(const duckdb::Value &value) {
 	return Int64GetDatum(pg_time);
 }
 
+static Datum
+ConvertTimeTzDatum(const duckdb::Value &value) {
+	duckdb::dtime_tz_t dt_tz = value.GetValue<duckdb::dtime_tz_t>();
+	const int64_t micros = dt_tz.time().micros;
+	const int32_t tz_offset = dt_tz.offset();
+
+	TimeTzADT *result = static_cast<TimeTzADT *>(palloc(sizeof(TimeTzADT)));
+	result->time = micros;
+	result->zone = -tz_offset;
```

Comment:
Why use `-` in this line? That's unexpected enough that I think it deserves a comment.

Reply:
Totally makes sense, add a comment on how pg/duckdb stores timezone.